spec/requests/api/v1.0/orders_rbac_spec.rb

```diff
@@ -52,27 +33,37 @@
   end
 
   context "Catalog User" do
-    let(:access_obj) { user_access_obj }
+    let(:catalog_access) { instance_double(Insights::API::Common::RBAC::Access, :scopes => %w[group]) }
```

Comment:
@eclarizio This should scopes => ['user']
Also we dont need ace_ids for Order only Portfolio goes into the ACE table